queries.sql

```diff
@@ -42,7 +42,7 @@ ROLLBACK;
 BEGIN;
 DELETE FROM animals WHERE date_of_birth > '2022-01-01';
 SAVEPOINT sp1;
-UPDATE aniamls SET weight_kg = weight_kg * -1;
+UPDATE animals SET weight_kg = weight_kg * -1;
```

Comment:
Please note that the queries are not present in your `queries.sql` file. Kindly make sure it is. 👍🏽